opsdroid/core.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, config=None, config_path=None):
         self.connector_tasks = []
         self.eventloop = asyncio.get_event_loop()
         if os.name != "nt":
-            for sig in (signal.SIGINT, signal.SIGTERM):
+            for sig in (signal.SIGINT, signal.SIGTERM, signal.SIGHUP):
```

Comment:
This isn't correct. The `SIGINT` and `SIGTERM` signals stop opsdroid running, where as we want `SIGHUP` to reload opsdroid.

This needs to be a separate block, which adds an event handler like `INT` and `TERM` but for a different method which reloads opsdroid. (I assume that method already exists).

Reply:
Hi @Cadair, I got your point and also understood that the SIGHUP handler should be added to the reload method (which already exists) but ain't quiet getting how to go about it. A little help would be appreciated:)